bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/connection/KNXConnection.java

```diff
@@ -433,6 +440,14 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
                 }
             }
 
+            String shouldUseNAT = (String) config.get("useNAT");
+            if (StringUtils.isNotBlank(shouldUseNAT)) {
+                sUseNAT = shouldUseNAT.equalsIgnoreCase("true");
+            }
+            else if (StringUtils.isBlank(shouldUseNAT)) {
```

Comment:
if isNotBlank is false, isn't isBlank true?  If so, the `if` above would be unnecessary?